testnet-contracts/contracts/Oracle.sol

```diff
@@ -241,15 +248,15 @@ contract Oracle {
                 }
         }
 
-        // Calculate if weighted signed power has reached threshold of weighted total power
-        uint256 weightedSignedPower = signedPower.mul(3);
-        uint256 weightedTotalPower = totalPower.mul(2);
-        bool hasReachedThreshold = weightedSignedPower >= weightedTotalPower;
+        // Prophecy must reach total signed power % threshold in order to pass consensus
```

Comment:
Maybe note that the `consensusThreshold` is a decimal multiplied by 100 which is why the `signedPower` is also being multiplied by 100

Reply:
https://github.com/cosmos/peggy/pull/160/commits/75f75ed2b188a9b9d9527ecd70ab4330cf4970b4